governance/polls/MIPs Monthly Governance Poll -  March 15, 2021.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Monthly MIPs Governance Poll - March 15, 2021
+summary: Signal your support or opposition for the monthly MIPs bundle for March
+discussion_link: https://forum.makerdao.com/t/weekly-mips-update-29/7000
+vote_type: Plurality Voting
+categories:
+   - Governance
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Monthly MIPs Governance Poll - March 15, 2021
+
+The Governance Facilitators have placed the MIP3: Monthly Governance Poll into the voting system. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, March 15 at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should abstain.**
+
+## Review
+
+This poll asks whether the bundled contents of this month's Governance Cycle should proceed to an Executive Vote to be held on March 22, 2021. The bundled contents of this poll consist of the following MIP Sets:
```

Comment:
Not all the groups below are MIP sets so I think this should revert to the template and just say "The bundled contents of this poll consist of:"

Reply:
I agree, thanks for pointing this out, @prose11 !